README.md

```diff
@@ -4,111 +4,136 @@ An API for validating survey schemas.
 
-In the eq-schema-validator directory, install node version manager (nvm) and node using the following commands:
+To run this you will need to add the `AJV_VALIDATOR_PORT` number to your .env file.
 
+Then run the Ajv server:
+``` shell
+make start-ajv
 ```
-brew install nvm
-nvm install
+
+To stop the Ajv server:
+``` shell
+make stop-ajv
 ```
 
-Install node dependencies:
+Running this returns either an empty json response when the questionnaire is valid, or a response containing an "errors" key. The errors are ordered by their path length and with first error message being the deepest path into the schema and should represent the best match for the questionnaire which has been posted.
 
-```
-npm install
-```
+### Testing and running against local schemas
 
-To run the ajv validator tests:
+By default, all schemas in the `tests/schemas/valid` directory will be evaluated as part of the unit tests. Any errors in these schemas will cause a failure. You can use this to run against local schemas by adding any schemas to this folder.
 
+To run the app's unit tests:
+``` shell
+make test-unit
 ```
+
+To run the Ajv validator tests:
+``` shell
 make test-ajv
 ```
 
-## Formatting json
+To run the app's unit tests and Ajv validator tests:
+``` shell
+make test
+```
 
-Run the following to format all json files in the schemas directory:
+#### Test the apps functionality against runner
 
+Spin it up with:
+``` shell
+make run
 ```
-make format
-````
 
-## Validating with ajv
+Then, in another terminal, navigate to a checked out copy of `https://github.com/ONSdigital/eq-questionnaire-runner` and run:
+``` shell
+make test
+```
 
-Also included is a node based version of the json schema validation which may be used during development to assist with
-debugging errors. This returns more errors than we'd currently like due to the way polymorphism works for each of our
-blocks.
+## Formatting json
 
-Run the ajv (server) based version of validator.
+Run the following to format all json files in the schemas directory:
 
+``` shell
+make format
 ```
-make start-ajv
-```
-To stop the ajv (server) based version of validator. You will need to add the AJV_VALIDATOR_PORT number to your .env file.
 
-```
-make stop-ajv
-```
+## Docker
 
-This returns either an empty json response when the questionnaire is valid, or a response containing an "errors" key.
-The errors are ordered by their path length and with first error message being the deepest path into the schema and
-should represent the best match for the questionnaire which has been posted.
+When PRs are merged in this repo there is a GitHub workflow that builds a Docker image of validator and then pushes to our GAR in GCP. This image can then be pulled down and run locally with Docker.
```

Comment:
Something about having two dockerfiles here?

Reply:
Also something about running them locally by hand like we do say in Runner README about mock sds and cir services?

Reply:
And something about it being used in Runner (`make run validator`, `make validate`, `docker-compose-schema-validator.yml` script)?